pyproject.toml

```diff
@@ -38,8 +38,11 @@ classifiers = [
     'Operating System :: MacOS',
 ]
 dependencies = [
-    "numpy>=1.21.2,<1.26.0; python_version < '3.12'",
-    "numpy>=1.26.0,<2.1.0; python_version >= '3.12' and python_version < '3.13'",
+#    "numpy>=1.21.2,<1.26.0; python_version < '3.12'",
+#    "numpy>=1.26.0,<2.1.0; python_version >= '3.12' and python_version < '3.13'",
+#    "numpy>=2.1.0; python_version >= '3.13'",
+    "numpy>=1.21.2,<1.26.0; python_version < '3.10'",
```

Comment:
@onuralpszr looks to me that python version here was off. it is updated in `develop` and this change make colab complain; can you double check what's happening here?